src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Operations/CosmosJobRecordWrapper.cs

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage;
```

Comment:
Is this change required? They only change I see in this file is the new using statement.